### PR TITLE
fixes unlimited explosion works 2.0: nazi boogaloo

### LINKED ISSUE #17334: mechahitler
ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

Now teleporting.

You are now an admin ghost. Think of yourself as an AI that doesn't show up anywhere and cannot speak. You can access any console or machine by standing next to it and clicking on it. Abuse of this privilege may result in hilarity or removal of your flags, so caution is recommended.

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Riccardo Benzoni says, "Oh."

(Follow) DEAD: Holly Burris mutters, "anyone else lagging?"

(Follow) Syd Walker says, "Evil does not die that easy"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) A Little Rogue states, "What happened."

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Ryan Jones says, "This sounds like meteors"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Syd Walker says, "Take this"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Sasha Tedrow asks, "Locker?"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) DEAD: Alana Mitchell cries, "it's hitler's fault"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Alexa Mason says, "Well"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Riccardo Benzoni exclaims, "Yes!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Tate Mcfall exclaims, "We gotta go!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Alexa Mason says, "I can't move"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) DEAD: Holly Burris cries, "i agree"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Syd Walker says, "If you see him stand to the right and yell now"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Gwenevere Mary says, "This is"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

Tony Gabagoul Pleads

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Gwenevere Mary says, "Really not good"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Blast Ripneck says, "Pods"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Two Hotdogs says, "Help me"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Bill Mason says, "Well this is a slow death"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: Explosion with size (1, 2, 3) in area Central Primary Hallway (226,224,1) (JMP)

ADMIN LOG: DrCelt/(Manu Yount) Has requested an immediate world restart via client side debugging tools.

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"

(Follow) Mecha Hitler says, "Eva, auf wiedersehen!"
	If a specific field doesn't apply, remove it!
	Anything inside tags like these is a comment and will not be displayed in the final issue.
	Be careful not to write inside them!
	Joke or spammed issues can and will result in punishment. Don't be a dick.
-->

#### Issue bingo

Please check whatever applies. More checkboxes checked increase your chances of not being yelled at by every contributor.

- [ ] Issue could be reproduced at least once
- [ ] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [ ] Issue happened in a recent (less than 7 days ago) round
- [ ] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by jknpj**
Same deal as #16111. Doing the same with Vox Armalis just to be sure.

Closes #17334

:cl:
 * bugfix: Fixes Mecha Hitler holocausting the whole server with infinite german explosions.

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/human/nazi.dm [hunk 1]
```diff
@@ -369,6 +369,6 @@
 	say("Eva, auf wiedersehen!")
 	ranged = 0
 	sleep(10)
-	explosion(loc,1,2,3)
 	new/obj/item/weapon/gun/gatling(loc)
 	qdel(src)
+	explosion(loc,1,2,3)
```
Comment (Intigracy) [inline code/modules/mob/living/simple_animal/hostile/human/nazi.dm]: won't this just delete the gatling gun you just spawned?

File: code/modules/mob/living/simple_animal/hostile/human/nazi.dm [hunk 2]
```diff
@@ -369,6 +369,6 @@
 	say("Eva, auf wiedersehen!")
 	ranged = 0
 	sleep(10)
-	explosion(loc,1,2,3)
-	new/obj/item/weapon/gun/gatling(loc)
 	qdel(src)
+	explosion(loc,1,2,3)
```
Comment (Intigracy) [inline code/modules/mob/living/simple_animal/hostile/human/nazi.dm]: And now that I think about this, pretty sure this is just going to cause an explosion in / spawn the gun in nullspace. Cast `loc` somewhere before the qdel.

**Review by Intigracy — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/vox.dm [hunk 3]
```diff
@@ -23,14 +23,12 @@
 	var/quills = 3
 
 /mob/living/simple_animal/vox/armalis/Die()
-
 	living_mob_list -= src
 	dead_mob_list += src
 	stat = DEAD
 	visible_message("<span class='danger'><B>[src] shudders violently and explodes!</B>","<span class='warning'>You feel your body rupture!</span></span>")
+	gib()
 	explosion(get_turf(loc), -1, -1, 3, 5)
```
Comment (Intigracy) [inline code/modules/mob/living/simple_animal/vox.dm]: Same here due to the gib().

**Review by Intigracy — APPROVED** [review]

**Comment by DeityLink** [comment]
Trying not to laugh out loud at work

**Pull request merged**